### DISCUSSION TITLE
Chaotic Events

### DISCUSSION THREAD

**Discussion by icefed** [community/community#137912] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

When I open https://github.com/, I get a bunch of confusing events, I just starred https://github.com/photoview/photoview, I don't know what kind of events these are coming from, because I've never received a pullrequest or issue event from a starred repository before. 

https://github.com/photoview/photoview/pull/1041

<img width="485" alt="Snipaste_2024-09-06_18-08-45" src="https://github.com/user-attachments/assets/99bafc83-9338-4e0c-8e98-e3c0c2e252d3">

<img width="818" alt="Snipaste_2024-09-06_18-17-21" src="https://github.com/user-attachments/assets/8ad1620b-297c-495a-8585-f0d6d9cdd843">

**Comment by LiteBrite82** [comment]
💬 Your Product Feedback Has Been Submitted 🎉
Thank you for taking the time to share your insights with us! Your feedback is invaluable as we build a better GitHub experience for all our users.

Here's what you can expect moving forward ⏩
- Your input will be carefully reviewed and cataloged by members of our product teams. 
- Other users may engage with your post, sharing their own perspectives or experiences. 
- GitHub staff may reach out for further clarification or insight. 

Where to look to see what's shipping 👀
- Read the [Changelog](https://github.blog/changelog/) for real-time updates on the latest GitHub features, enhancements, and calls for feedback.
- Explore our [Product Roadmap](https://github.com/orgs/github/projects/4247), which details upcoming major releases and initiatives.

What you can do in the meantime 💻
- Upvote and comment on other user feedback Discussions that resonate with you.
- Add more information at any point! Useful details include: use cases, relevant labels, desired outcomes, and any accompanying screenshots.

As a member of the GitHub community, your participation is essential. While we can't promise that every suggestion will be implemented, we want to emphasize that your feedback is instrumental in guiding our decisions and priorities.

Thank you once again for your contribution to making GitHub even better! We're grateful for your ongoing support and collaboration in shaping the future of our platform. ⭐

**Comment by icefed** [comment]
<img width="463" alt="image" src="https://github.com/user-attachments/assets/cc5c9ef4-4b94-4802-97ab-ed7d670579ba">

<img width="964" alt="image" src="https://github.com/user-attachments/assets/bf8dfd1c-92a3-4c7d-be4a-39ed65e0ef8d">


This happens again, other repos I stared don't have those events, is this a bug or personal setting issue? how can I disable it.

**Reply by AESMatias** [reply]
Greetings @icefed, have you checked if you have this option enabled?
![image](https://github.com/user-attachments/assets/97e3415a-ba7a-4799-8a24-8e5491bb9f0f)
I hope it isn't a bug (i'm not brave enough to star* the repo), but if the problem persist, you can always contact the GitHub support by submitting a ticket: https://support.github.com/request

**Reply by icefed** [reply]
Thanks, I checked this, I just stared repo, never used `watch` and `subscribe` to https://github.com/photoview/photoview.
<img width="625" alt="image" src="https://github.com/user-attachments/assets/20a5c05a-e916-416f-8d7d-cdd6240ece06">
<img width="838" alt="image" src="https://github.com/user-attachments/assets/40732254-2c55-4d37-ad3f-669828d2e93a">

**Reply by icefed** [reply]
those events all `labeled a pull request` from `https://github.com/photoview/photoview`.

**Comment by icefed** [comment]
again, I have to unstar https://github.com/photoview/photoview.

another problem, I lost some events I have seen before, the latest release https://github.com/jordanbaird/Ice/releases, but I only can see `Ice 0.11.2` in the first page of `https://github.com/`.
<img width="714" alt="image" src="https://github.com/user-attachments/assets/7ed2c624-12dd-4b75-b28b-7f185acca0e2">

**Reply by icefed** [reply]
the lost events come back, what's wrong?
